src/components/panes/TargetPane.jsx

```diff
@@ -21,23 +22,39 @@ export default class TargetPane extends PaneBase {
     renderContent() {
         let target = this.props.call.get('target');
 
-        let info = [
+        const addrFields = ADDR_FIELDS.filter(field => target.get(field)).map(field => 
+            <span key={ field }>
+                { target.get(field) }
+            </span>
+        );
+
+         let info = [
             <li key="id" className="TargetPane-infoId">
                 { target.get('ext_id') || '-' }</li>,
             <li key="email" className="TargetPane-infoEmail">
-                { target.get('email') }</li>
+                { target.get('email') }</li>,
+            <li key="address" className="TargetPane-infoAddress">
+                { addrFields }
+            </li>,
         ];
-        if (target.get('city')) {
-            info.push(
-                <li key="city" className="TargetPane-infoCity">
-                    { target.get('city') }</li>
-            )
+
+
+        if(target.has('person_fields')) {
+            const customFields = target.get('person_fields')
+                .map(field => 
+                    <li className="TargetPane-personField" key={field}>
+                        <span className="TargetPane-personFieldLabel">{ field.get('title')}</span>
+                        <span className="TargetPane-personFieldValue">{ field.get('value') }</span>
+                    </li>
+                ).toArray();
+
+            info = info.concat(customFields);
         }
 
         let map = null;
         if (target.get('zip_code') && target.get('city')) {
             let qs = querystring.stringify({
-                center: target.get('zip_code') + '+' + target.get('city'),
+                center: target.get('street_address') + '+' + target.get('zip_code') + '+' + target.get('city'),
```

Comment:
This results in `undefined 21420 Malmö` if street address is not exposed, and causes the map to render in the middle of the sea somewhere.